storage/series.go

```diff
@@ -0,0 +1,182 @@
+package storage
+
+import (
+	"math"
+	"sort"
+
+	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/prometheus/prometheus/tsdb/chunkenc"
+	"github.com/prometheus/prometheus/tsdb/chunks"
+	"github.com/prometheus/prometheus/tsdb/tsdbutil"
+)
+
+type ConcreteSeries struct {
+	labels           labels.Labels
+	SampleIteratorFn func() chunkenc.Iterator
+	ChunkIteratorFn  func() chunks.Iterator
+}
+
+func NewTestSeries(lset labels.Labels, samples ...[]tsdbutil.Sample) Series {
```

Comment:
this is only for testing right? so why not put in a test package or a test file?

Reply:
Because we want to reuse this across all unit test, even those not in this package.